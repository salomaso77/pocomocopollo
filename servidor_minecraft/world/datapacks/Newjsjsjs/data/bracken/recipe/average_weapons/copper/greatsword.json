{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		"## ",
		"/# "
	],
	"key": {
		"#": "minecraft:copper_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:max_damage": 175,
			"minecraft:item_name": "Copper Greatsword",
			"minecraft:item_model": "bracken:average_weapons/copper/greatsword",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "6.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:copper_greatsword",
					"amount": 5.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:copper_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:copper_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}