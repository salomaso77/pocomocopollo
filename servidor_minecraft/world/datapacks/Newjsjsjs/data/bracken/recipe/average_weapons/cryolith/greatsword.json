{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:dune_armor_trim_smithing_template",
	"base": "minecraft:iron_sword",
	"addition": "minecraft:polar_bear_spawn_egg",
	"result": {
		"id": "minecraft:iron_sword",
		"components": {
			"minecraft:max_damage": 200,
			"minecraft:item_name": "Cryolith Greatsword",
			"minecraft:item_model": "bracken:average_weapons/cryolith/greatsword",
			"minecraft:enchantments": {
				"minecraft:vanishing_curse": 1,
				"bracken:freezing_fix": 1,
				"bracken:fragility_curse": 1
			},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "11 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:cryolith_greatsword",
					"amount": 10,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:cryolith_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:cryolith_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}