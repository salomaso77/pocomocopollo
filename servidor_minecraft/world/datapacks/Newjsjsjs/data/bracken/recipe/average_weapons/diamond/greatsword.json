{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		"## ",
		"/# "
	],
	"key": {
		"#": "minecraft:diamond",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:diamond_sword",
		"components": {
			"minecraft:item_name": "Diamond Greatsword",
			"minecraft:item_model": "bracken:average_weapons/diamond/greatsword",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "8 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:diamond_greatsword",
					"amount": 7,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:diamond_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:diamond_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}