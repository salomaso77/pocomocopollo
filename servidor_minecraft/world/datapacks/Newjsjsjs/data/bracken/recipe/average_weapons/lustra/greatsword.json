{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:dune_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:sheep_spawn_egg",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:max_damage": 3000,
			"minecraft:enchantment_glint_override": true,
			"minecraft:item_name": "Lustra Greatsword",
			"minecraft:item_model": "bracken:average_weapons/lustra/greatsword",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:lustra_greatsword",
					"amount": 4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:lustra_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:lustra_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}