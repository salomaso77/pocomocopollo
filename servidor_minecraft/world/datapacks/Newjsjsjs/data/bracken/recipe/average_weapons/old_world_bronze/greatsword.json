{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:dune_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:cat_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Old World Bronze Greatsword",
			"minecraft:item_model": "bracken:average_weapons/old_world_bronze/greatsword",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "9.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_greatsword",
					"amount": 8.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:netherite_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}