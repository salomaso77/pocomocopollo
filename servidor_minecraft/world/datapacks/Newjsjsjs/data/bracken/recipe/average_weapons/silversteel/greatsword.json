{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:dune_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:wolf_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Silversteel Greatsword",
			"minecraft:enchantments": {
				"bracken:silvered_fix": 1
			},
			"minecraft:enchantment_glint_override": false,
			"minecraft:item_model": "bracken:average_weapons/silversteel/greatsword",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "10 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:silversteel_greatsword",
					"amount": 9,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:silversteel_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:silversteel_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}