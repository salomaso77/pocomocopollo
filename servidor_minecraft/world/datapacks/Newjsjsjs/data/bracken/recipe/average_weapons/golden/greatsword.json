{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		"## ",
		"/# "
	],
	"key": {
		"#": "minecraft:gold_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "Golden Greatsword",
			"minecraft:item_model": "bracken:average_weapons/golden/greatsword",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+1 Attack Range","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_greatsword",
					"amount": 4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_greatsword",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:entity_interaction_range",
					"id": "bracken:golden_greatsword",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}